src/business-logic/services/database-change-ingest-service.ts

```diff
@@ -33,11 +36,13 @@ export class DatabaseChangeIngestService implements IngestChangeService {
     timelineRepository: TimelineRepository,
     timelineBuilder: TimelineBuilder,
     eventEmitter: RundownEventEmitter,
+    mediaEventEmitter: MediaEventEmitter,
```

Comment:
Since there now is more than one event emitter here, it would be a good idea to update the name of the above `eventEmitter` to `rundownEventEmitter`

Reply:
Changed.